protobuf2.5/plan.sh

```diff
@@ -0,0 +1,14 @@
+pkg_name=protobuf
```

Comment:
That seems a little counter-intuitive - I am expecting to use the origin/pkg/version naming scheme.